common/djangoapps/xblock_django/tests/test_user_service.py

```diff
@@ -112,3 +116,16 @@ def test_get_anonymous_user_id_returns_id_for_existing_users(self):
         )
 
         self.assertEqual(anonymous_user_id, anon_user_id)
+
+    def test_external_id(self):
+        """
+        Tests that external ids differ based on type.
+        """
+        ExternalIdType.objects.create(name='test1')
```

Comment:
As a good practice, we should always include a description when creating a type.